test/test_testing.py

```diff
@@ -4002,7 +4053,41 @@ def test_send_signal(self):
 class _PebbleStorageAPIsTestMixin:
     # Override this in classes using this mixin.
     # This should be set to any non-empty path, but without a trailing /.
-    prefix = None
+    prefix: str = None  # type: ignore
```

Comment:
Is the `= None` required here and below? I think you can drop the `= None` and the `type: ignore`.

Reply:
Nice, didn't think of that.